src/openrct2/object/ImageTable.cpp

```diff
@@ -97,6 +368,71 @@ void ImageTable::Read(IReadObjectContext* context, OpenRCT2::IStream* stream)
     }
 }
 
+void ImageTable::ReadJson(IReadObjectContext* context, json_t& root)
+{
+    Guard::Assert(root.is_object(), "ImageTable::ReadJson expects parameter root to be object");
+
+    if (context->ShouldLoadImages())
+    {
+        // First gather all the required images from inspecting the JSON
+        std::vector<std::unique_ptr<RequiredImage>> allImages;
+        auto jsonImages = root["images"];
+
+        // json_array_foreach(jsonImages, i, el)
```

Comment:
Does it make sense to keep this comment?

Reply:
Commented-out code should be taken out unless there is a damn good reason why it's there (and even then it should be taken out).

Reply:
I doubt it is intentional, likely commented out when writing the new code and then forgot to delete it afterwards.